Dockerfile

```diff
@@ -1,8 +1,26 @@
 # syntax=docker/dockerfile:1
+FROM ghcr.io/linuxserver/baseimage-alpine:3.19 as buildstage
+
+ARG MOD_VERSION
+
+RUN apk add --no-cache curl jq && \
```

Comment:
Our 3.19 baseimage includes jq and curl so this line can be removed

Reply:
done